### PR TITLE
1307 1 webpack setup

### PR DESCRIPTION
**Pull request by kkyrylov**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = {
+    // entry point of application
+    entry: './src/index.js',
+    // output path for bundled files
+    output: {
+        // path for the built project code
+        path: path.resolve(__dirname, 'distributable'),
```
Comment (sarhan-azizov) [inline webpack.config.js]: path: path.resolve(__dirname, 'dist'),

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        // name for the output file
+        filename: 'my-first-dist.bundle.js'
```
Comment (sarhan-azizov) [inline webpack.config.js]: filename: 'index.js'

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "test"
+    "test": "test",
```
Comment (sarhan-azizov) [inline package.json]: "test": "jest",

Comment (kkyrylov) [reply package.json]: sry, that came with npm init ) my bad

File: webpack.config.js (hunk 1 above)
Comment (kkyrylov) [reply webpack.config.js]: changing name to dist

File: webpack.config.js (hunk 2 above)
Comment (kkyrylov) [reply webpack.config.js]: changing name to index.js

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**